src/appengine/handlers/testcase_detail/show.py

```diff
@@ -601,6 +603,12 @@ def get_testcase_detail(testcase):
           reproduction_help_url,
       'is_local_development':
           environment.is_running_on_app_engine_development(),
+      'fuzzer_display': {
```

Comment:
maybe add a as_dict() ? so this can be used in more places in the future?

Reply:
Done, let me use vars()